pallets/creditcoin/src/ocw/tasks.rs

```diff
@@ -1,4 +1,4 @@
-pub mod collect_coins;
+//pub mod collect_coins;
```

Comment:
IMO just delete it instead of commenting it out. The history is still going to be available in git so the removal can be as easily reverted.

Reply:
still wip, hence the draft pr, will cleanup soon